linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
ES6 should be the standard, even if it's not written anywhere. It's the same as the rules you wrote for rubocop, they became a standard once you wrote them and people agreed. If you don't agree that ES6 should be the standard, then we can discuss that also.

I think two configurations are enough. This is the stack we should be moving towards. Having many configurations will require that we keep them all in sync. The configuration can always be changed to match each project's needs, but there must be one default.